agasc/tests/test_agasc.py

```diff
@@ -37,9 +37,15 @@
     assert os.path.exists(ascrc_file)
     ascds_env = Ska.Shell.getenv('source {} -r release'.format(ascrc_file), shell='tcsh')
     assert 'ASCDS_BIN' in ascds_env
+    # Make a mp_get_agasc cmd line that just returns that command's status
+    cmd = 'mp_get_agasc -r 10 -d 20 -w 0.01 >& /dev/null; echo $?'
+    lines = Ska.Shell.tcsh(cmd, env=ascds_env)
+    # Check command status is 0
+    assert lines[0] == "0"
```

Comment:
Do we have something nicer in the Ska.Shell tools to deal with status?  I thought we must but then couldn't figure it out.